src/components/LineChart/LineChart.jsx

```diff
@@ -371,12 +380,14 @@ const LineChart = createClass({
 			yAxisTitleColor,
 			yAxisMin,
 			yAxisTooltipFormatter,
+			yAxisTooltipDataFormatter,
```

Comment:
hm, that's weird, you should be able to as long as the one you're setting it to is above the one you're setting the default on